src/utils/plotSpecs/generateTrajectoryAnalysisSpec.js

```diff
@@ -59,6 +59,7 @@ const generateBaseSpec = (
   embeddingData,
   viewState,
   numClusters,
+  method,
```

Comment:
Why can't we use the config and need to pass a separate parameter now?

Reply:
Using this to generate the default axis names, instead of dispatching a redux action updateEmbeddingAxesIfNecessary every time, like it was done before.